### PR TITLE
tikv-control: make a parameter in right places

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9886
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-control.md [hunk 1]
```diff
@@ -229,6 +229,7 @@ Use the `compact` command to manually compact data of each TiKV. If you specify
 
 - Use the `--host` option to specify the TiKV that needs to perform compaction.
 - Use the `-d` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
```
Comment (Connor1996) [inline tikv-control.md:231]: remove this

Comment (Oreoxmt) [reply tikv-control.md]: Done in https://github.com/pingcap/docs/pull/9549/commits/a670d190157297acd121dff245bfe56350a95939

File: tikv-control.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ The properties can be used to check whether the Region is healthy or not. If not
 Use the `compact` command to manually compact data of each TiKV. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key.
 
 - Use the `--host` option to specify the TiKV that needs to perform compaction.
-- Use the `-d` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
```
Comment (Connor1996) [inline tikv-control.md:231]: sorry, I mistake the option, this line should be reserved.

File: tikv-control.md [hunk 3]
```diff
@@ -228,7 +228,7 @@ The properties can be used to check whether the Region is healthy or not. If not
 Use the `compact` command to manually compact data of each TiKV. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key.
```
Comment (Connor1996) [inline tikv-control.md:228]: I refresh the whole description, please update
---------------------------------------------------
### Compact data of each TiKV manually

Use the `compact` command to manually compact data of each TiKV. 

- Use the `--from` and `--to` options to sepecify the compact range in the form of escaped raw key. If not set, the whole range will be compacted.
- Use the `--region` option to compact range of a specific region. If set, `--from` and `--to` will be ignored.
- Use the `--db` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
- Use the `--threads` option allows you to specify the concurrency for the TiKV compaction and its default value is `8`. Generally, a higher concurrency comes with a faster compaction speed, which might yet affect the service. You need to choose an appropriate concurrency count based on your scenario.
- Use the `--bottommost` option to include or exclude the bottommost files when TiKV performs compaction. The value options are `default`, `skip`, and `force`. The default value is `default`.
    - `default` means that the bottommost files are included only when the Compaction Filter feature is enabled.
    - `skip` means that the bottommost files are excluded when TiKV performs compaction.
    - `force` means that the bottommost files are always included when TiKV performs compaction.


- For local mode
{{< copyable "shell-regular" >}}
```shell
$ tikv-ctl --data-dir /path/to/tikv compact --db kv
success!
```
- For remote mode
{{< copyable "shell-regular" >}}
```shell
$ tikv-ctl --host ip:port compact --db kv
success!
```

**Comment by Oreoxmt** [comment]
@Connor1996 @hicqu PTAL

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tikv-control.md [hunk 4]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
 
 ### Compact data of each TiKV manually
 
-Use the `compact` command to manually compact data of each TiKV. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key.
+Use the `compact` command to manually compact data of each TiKV.
 
-- Use the `--host` option to specify the TiKV that needs to perform compaction.
-- Use the `-d` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
+- Use the `--from` and `--to` options to specify the compact range in the form of escaped raw key. If not set, the whole range will be compacted.
```
Comment (TomShawn) [inline tikv-control.md:235]:
```suggestion
- Use the `--from` and `--to` options to specify the compaction range in the form of escaped raw key. If not set, the whole range will be compacted.
```

File: tikv-control.md [hunk 5]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
...
+- Use the `--region` option to compact range of a specific region. If set, `--from` and `--to` will be ignored.
```
Comment (TomShawn) [inline tikv-control.md:236]: Could you please add this to Chinese accordingly?
```suggestion
- Use the `--region` option to compact the range of a specific region. If set, `--from` and `--to` will be ignored.
```

File: tikv-control.md [hunk 6]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
...
+- Use the `--db` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
```
Comment (TomShawn) [inline tikv-control.md:237]: Please add this to Chinese.  Thanks~

File: tikv-control.md [hunk 7]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
...
 - Use the `--threads` option allows you to specify the concurrency for the TiKV compaction and its default value is `8`. Generally, a higher concurrency comes with a faster compaction speed, which might yet affect the service. You need to choose an appropriate concurrency count based on your scenario.
```
Comment (TomShawn) [inline tikv-control.md:238]: [This line](https://github.com/pingcap/docs-cn/pull/9886/files#diff-ff89854f38035fc717cda9246e345cffc32f58be987165f3529f06b3a4586065R289) is missing in the translation. Please add it here.

File: tikv-control.md [hunk 8]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
...
+```shell
+$ tikv-ctl --data-dir /path/to/tikv compact --db kv
+success!
+```
+
+- To compact data in the remote mode, use the following command:
+
+```shell
+$ tikv-ctl --host ip:port compact --db kv
 success!
```
Comment (TomShawn) [inline tikv-control.md:255]: Sync this to Chinese so long as the tech owner approves.

File: tikv-control.md (hunk 7 above)
Comment (Oreoxmt) [reply tikv-control.md:238]: The `--data-dir` option is removed in https://github.com/pingcap/docs-cn/pull/11026

File: tikv-control.md (hunk 6 above)
Comment (Oreoxmt) [reply tikv-control.md:237]: https://github.com/pingcap/docs-cn/pull/11026

File: tikv-control.md [hunk 9]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
...
     - `force` means that the bottommost files are always included when TiKV performs compaction.
 
-```bash
-$ tikv-ctl --data-dir /path/to/tikv compact -d kv
+- To compact data in the local mode, use the following command:
+
+```shell
+$ tikv-ctl --data-dir /path/to/tikv compact --db kv
+success!
+```
```
Comment (TomShawn) [inline tikv-control.md:249]:
````suggestion
    ```shell
    tikv-ctl --data-dir /path/to/tikv compact --db kv
    success!
    ```
````

File: tikv-control.md [hunk 10]
```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
...
+
+- To compact data in the remote mode, use the following command:
+
+```shell
+$ tikv-ctl --host ip:port compact --db kv
 success!
 ```
```
Comment (TomShawn) [inline tikv-control.md:256]:
````suggestion
    ```shell
    tikv-ctl --host ip:port compact --db kv
    success!
    ```
````

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: tikv-control.md [hunk 11]
```diff
@@ -230,20 +230,30 @@ The properties can be used to check whether the Region is healthy or not. If not
...
 
-```bash
-$ tikv-ctl --data-dir /path/to/tikv compact -d kv
-success!
-```
+- To compact data in the local mode, use the following command:
+
+    ```shell
+    tikv-ctl --data-dir /path/to/tikv compact --db kv
+    success!
```
Comment (TomShawn) [inline tikv-control.md:248]:
```suggestion
```

File: tikv-control.md [hunk 12]
```diff
@@ -230,20 +230,30 @@ The properties can be used to check whether the Region is healthy or not. If not
...
+    ```
+
+- To compact data in the remote mode, use the following command:
+
+    ```shell
+    tikv-ctl --host ip:port compact --db kv
+    success!
```
Comment (TomShawn) [inline tikv-control.md:255]:
```suggestion
```

**Pull request merged**